### PR TITLE
Add package search exact match feature

### PR DESCRIPTION
**Pull request by anaggh**
Before:

before

After:

after

### CODE REVIEW

**Comment by ericonr** [comment]
Maybe would be nice to include this in the same result list? Or at least align the columns.

**Comment by anaggh** [comment]
It uses the same styles as the 2nd table. 

The reason the width is different is because some of the results in the 2nd table have a long description.

**Comment by ericonr** [comment]
so for style I think we should merge into a single table, maybe? What do you think?

**Comment by anaggh** [comment]
Okay, it now looks like this

afterNew

File: assets/js/voidsearch.js [hunk 1]
```diff
@@ -100,6 +101,28 @@
...
             table.append(tooManyNotice());
         }
+
+        if (packagesExactMatch.length >= 1) {
+            $('#voidSearch_result tr:first').after(
+                packagesExactMatch.map((p) => {
+                    let ghSlug = p.name.replace(/-(?:32bit|dbg)$/, "");
+                    p.anchor = "<a href=\"https://github.com/void-linux/void-packages/tree/master/srcpkgs/"
+                        + ghSlug + "\" target=\"_blank\" title=\"View on GitHub\">" + p.name + "</a>";
+                    p.filename_size = typeof(p.filename_size) == 'number' ? formatSize(p.filename_size) : p.filename_size;
```
Comment (ericonr) [inline assets/js/voidsearch.js:111]: Maybe make this into a function, if it can avoid the repetition? (Note I don't really know JS)

**Comment by ericonr** [comment]
I really like that! Maybe it could be bold as well (I don't know if others agree with me).

Can you please squash your commits as well?

**Comment by anaggh** [comment]
Done!

File: assets/js/voidsearch.js (hunk 1 above)
Comment (anaggh) [reply assets/js/voidsearch.js]: Added `transformPackageForPackageCell(p)`

**Comment by anaggh** [comment]
I created a new branch and updated the site from bootstrap 3.x to 5.x. If you are interested, I ll open a pull request. https://github.com/anaggh/void-linux.github.io/commit/fa001156377054a78412caf975a7bfa16695df4e

Peek 2021-01-22 23-44

**Comment by ericonr** [comment]
Hi! Why did you close this? :(

**Comment by ericonr** [comment]
Updating bootstrap also seems pretty reasonable.

**Comment by anaggh** [comment]
I moved back to Debian on desktop (already using it on servers) since I am seeing a lot of open source developers/projects disappearing (probably dead /corona) . This is concerning.

* https://github.com/void-linux/void-infrastructure/pull/97

**Comment by ericonr** [comment]
Regarding that specific issue, it was opened because the user hasn't been active for a long time. The rest of the Void team is very much active.

**Pull request merged**